src/Swarm/TUI/View.hs

```diff
@@ -588,6 +588,7 @@ helpWidget theSeed mport =
     , ("Ctrl-z", "decrease speed")
     , ("Ctrl-w", "increase speed")
     , ("Ctrl-q", "quit the current scenario")
+    , ("Ctrl-r", "collapse/expand REPL")
```

Comment:
Note we already use ctrl-r for reverse search in the repl, so we'll have to pick a different shortcut for this.

Reply:
Thanks for the heads up!  :+1:

Reply:
changed to "Ctrl-k"

Reply:
Ctrl-k is in use also, it kills text to the end of the line (just like in emacs or on the command line).  Maybe Ctrl-SPC?  I don't know if that will work in terminals.  I would also suggest Ctrl-M (for Minimize) but I seem to recall that just generates a `Return` key event... Ctrl-n, perhaps?  Grasping at straws here...